atlasdb-workload-server-antithesis/src/test/java/AntithesisDockerTest.java

```diff
@@ -48,32 +48,42 @@ public class AntithesisDockerTest {
 
     @Test
     public void workloadServerHasRunDesiredWorkflowsSuccessfully() {
-        AtomicBoolean hasRunSuccessfully = new AtomicBoolean(false);
-
         String successMessage = "Finished running desired workflows successfully";
         String failureMessage = "Workflow will now exit.";
 
-        Awaitility.await()
-                .atMost(Duration.ofMinutes(5))
-                .pollInterval(Duration.ofSeconds(10))
-                .until(() -> {
-                    OutputStream logStream = new ByteArrayOutputStream();
-                    dockerComposeExtension
-                            .dockerComposeExecutable()
-                            .execute("logs", "workload-server")
-                            .getInputStream()
-                            .transferTo(logStream);
+        String logs = waitUntilDockerComposeSucceededOrFailedAndGetLogs(successMessage, failureMessage);
+        assertThat(logs).contains(successMessage);
```

Comment:
Updating the way this test is structured so we actually have full access to the logs when it fails.